qa/rpc-tests/getblockexpanded.py

```diff
@@ -0,0 +1,260 @@
+
+        connect_nodes_bi(self.nodes,0,1)
+
+        self.is_network_split=False
+        self.sync_all()
+
+    def run_test(self):
+
+        #amounts
+        creation_amount = Decimal("50")
+        bwt_amount = Decimal("5")
+        tAddr1 = self.nodes[1].getnewaddress()
+        node1Addr = self.nodes[1].validateaddress(tAddr1)['address']
+        self.nodes[0].generate(MINIMAL_SC_HEIGHT)
+        self.sync_all()
+        
+        ########### Create the sidechain ##################
+        print("########### Create the sidechain ##################")
+        
+        mcTest = CertTestUtils(self.options.tmpdir, self.options.srcdir)
+        vk = mcTest.generate_params("sc1")
+        constant = generate_random_field_element_hex()
+
+        ret = self.nodes[0].dep_sc_create(EPOCH_LENGTH, "dada", creation_amount, vk, "", constant)
+        creating_tx = ret['txid']
+        scid = ret['scid']
+        scid_swapped = str(swap_bytes(scid))
+        self.sync_all()
+
+        decoded_tx = self.nodes[0].getrawtransaction(creating_tx, 1)
+        assert_equal(scid, decoded_tx['vsc_ccout'][0]['scid'])
+
+        sc_creation_block_hash = self.nodes[0].generate(1)[0]
+        sc_creation_block = self.nodes[0].getblock(sc_creation_block_hash)
+        self.sync_all()
+
+        #Advance for 1 Epoch
+        self.nodes[0].generate(EPOCH_LENGTH)
+        self.sync_all()
+
+        ########### Mine Certificate 1 with quality = 5 ##################
+        print("########### Mine Certificate 1 with quality = 5 ##################")
+
+        epoch_number, epoch_cum_tree_hash = get_epoch_data(scid, self.nodes[0], EPOCH_LENGTH)
+        quality = 5
+        proof = mcTest.create_test_proof(
+            "sc1", scid_swapped, epoch_number, quality, MBTR_SC_FEE, FT_SC_FEE, epoch_cum_tree_hash, constant, [node1Addr], [bwt_amount])
+
+        amount_cert_1 = [{"address": node1Addr, "amount": bwt_amount}]
+
+        self.nodes[0].sc_send_certificate(scid, epoch_number, quality,
+            epoch_cum_tree_hash, proof, amount_cert_1, FT_SC_FEE, MBTR_SC_FEE, CERT_FEE)
+        self.sync_all()
+        maturityHeight = sc_creation_block["height"]+(EPOCH_LENGTH*2)+EPOCH_LENGTH*0.2 - 1
+
+        #Add to mempool Certificate 2 with quality = 7
+        print("########### Add to mempool Certificate 2 with quality = 7 ##################")
+        quality = 7  
+        bwt_amount2 = Decimal("7")      
+        proof = mcTest.create_test_proof(
+            "sc1", scid_swapped, epoch_number, quality, MBTR_SC_FEE, FT_SC_FEE, epoch_cum_tree_hash, constant, [node1Addr], [bwt_amount2])
+
+        amount_cert_2 = [{"address": node1Addr, "amount": bwt_amount2}]
+
+        self.nodes[0].sc_send_certificate(scid, epoch_number, quality,
+            epoch_cum_tree_hash, proof, amount_cert_2, FT_SC_FEE, MBTR_SC_FEE, CERT_FEE)
+        self.sync_all()
+
+        #Mine a block
+        self.nodes[0].generate(1)[0]
```

Comment:
Accessing the element `[0]` seems useless.
```suggestion
        self.nodes[0].generate(1)
```